ammo-be/src/routers/bulletRouter/index.ts

```diff
@@ -13,14 +13,26 @@ bulletRouter.post(
         try {
             const connectorRequest: ConnectorRequest = { ...req.body.data };
 
-            const { error, value } = bulletSchema.validate(connectorRequest);
-            if (error) throw new Error(error.details[0].message);
+            if (isRecording) {
```

Comment:
Do the opposite to avoid lots of tabs : 'fail fast'. Handle isRecording = false first.